beetsplug/discogs.py

```diff
@@ -200,7 +200,7 @@ def get_albums(self, query):
         query = re.sub(r'(?u)\W+', ' ', query).encode('ascii', "replace")
         # Strip medium information from query, Things like "CD1" and "disk 1"
         # can also negate an otherwise positive result.
-        query = re.sub(br'(?i)\b(CD|disc)\s*\d+', '', query)
+        query = re.sub(b'(?i)\b(CD|disc)\s*\d+', '', query)
```

Comment:
Agreed—it looks to me like that empty string literal should almost certainly be a `b''`.